src/DateRangePicker.elm

```diff
@@ -111,8 +113,23 @@ type alias Config =
     }
 
 
+{-| ActionButtons labels configuration:
+
+  - `close`: Button, which will close daterange-picker
+  - `clear`: Button, which will clear input string
+  - `apply`: Button, which will set new daterange
+
+-}
+type alias ActionButtons =
+    { close : String
+    , clear : String
+    , apply : String
```

Comment:
There are other texts to be translated : https://github.com/allo-media/elm-daterange-picker/blob/master/src/DateRangePicker.elm#L535-L538

These should be added to this record, which may be renamed more appropriately :)

Reply:
I added another record and move type aliases to better place, I think.